src/components/main-form.js

```diff
@@ -10,26 +10,29 @@ import styled, { css } from "react-emotion";
 import { Input } from 'reactstrap';
 import { FV, PMT } from 'formulajs/lib/financial';
 
+// DEFAULT VALUES
 
 const COMPOUND_FREQUENCY = 12;
 
-const o = {
+const DEFAULT_OPTION = {
   interestAmt: 0,
   interestRate: 0,
   term: 0,
   pmt: 0,
   fv: 0
 };
 
-let o1 = Object.assign({}, o);
-let o2 = Object.assign({}, o);
+let o1 = Object.assign({}, DEFAULT_OPTION);
+let o2 = Object.assign({}, DEFAULT_OPTION);
```

Comment:
I also prefer having `o1` be `option1`, easier to read!

Reply:
np